pkg/apiserver/start_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2025 BWI GmbH and Artifact Conduit contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package apiserver_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"go.opendefense.cloud/arc/pkg/apiserver"
+	"k8s.io/apimachinery/pkg/util/version"
+)
+
+var _ = Describe("ARCVersionToKubeVersion", func() {
```

Comment:
Uff, not sure if we should test this, as the current version conversion simply carried over from the sample. 

@jastBytes Any preferences? Strictly speaking test doesn't hurt, but is testing code that needs to be revisited....